spec/features/event_image_spec.rb

```diff
@@ -19,20 +18,20 @@
     it "should correctly set an image even if first validation fails" do
       fill_in "Maximale Teilnehmerzahl", :with => nil
       attach_file "event[custom_image]", File.join(Rails.root + 'spec/testfiles/image_upload_test.png')
-      click_button I18n.t(".events.form.create")
+      click_button "create"
 
       fill_in "Maximale Teilnehmerzahl", :with => 25
-      click_button I18n.t(".events.form.create")
+      click_button "create"
       expect(Event.last.image).to be_present
     end
 
     it "should not change the image if validation for a new image fails" do
-      click_button I18n.t(".events.form.create")
+      click_button "create"
       previous_image = Event.last.image
 
       visit edit_event_path(Event.last)
       attach_file "event[custom_image]", File.join(Rails.root + 'spec/testfiles/too_small_image.png')
-      click_button I18n.t('.events.form.update')
+      click_button "update"
```

Comment:
Okay. Since this never lets a line exceed 80 chars or even create a new line, I'm in favor of the I18n version, but I'd like more opinions on this here @hpi-schuelerklub/swt2